features/viewfolder/impl/src/main/kotlin/io/element/android/features/viewfolder/impl/file/ViewFileView.kt

```diff
@@ -0,0 +1,206 @@
+import io.element.android.libraries.androidutils.system.copyToClipboard
+import io.element.android.libraries.designsystem.components.button.BackButton
+import io.element.android.libraries.designsystem.icons.CompoundDrawables
+import io.element.android.libraries.designsystem.preview.ElementPreview
+import io.element.android.libraries.designsystem.preview.PreviewsDayNight
+import io.element.android.libraries.designsystem.theme.aliasScreenTitle
+import io.element.android.libraries.designsystem.theme.components.Icon
+import io.element.android.libraries.designsystem.theme.components.IconButton
+import io.element.android.libraries.designsystem.theme.components.Scaffold
+import io.element.android.libraries.designsystem.theme.components.Text
+import io.element.android.libraries.designsystem.theme.components.TopAppBar
+import io.element.android.libraries.ui.strings.CommonStrings
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ViewFileView(
+    state: ViewFileState,
+    onBackPressed: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Scaffold(
+        modifier = modifier,
+        topBar = {
+            TopAppBar(
+                navigationIcon = {
+                    BackButton(onClick = onBackPressed)
+                },
+                title = {
+                    Text(
+                        text = state.name,
+                        style = ElementTheme.typography.aliasScreenTitle,
+                    )
+                },
+                actions = {
+                    IconButton(
+                        onClick = {
+                            state.eventSink(ViewFileEvents.Share)
+                        },
+                    ) {
+                        Icon(
+                            resourceId = CompoundDrawables.ic_share_android,
+                            contentDescription = stringResource(id = CommonStrings.action_share),
+                        )
+                    }
+                    IconButton(
+                        onClick = {
+                            state.eventSink(ViewFileEvents.SaveOnDisk)
+                        },
+                    ) {
+                        Icon(
+                            resourceId = CompoundDrawables.ic_download,
+                            contentDescription = stringResource(id = CommonStrings.action_save),
+                        )
+                    }
+                }
+            )
+        },
+        content = { padding ->
+            Column(
+                modifier = Modifier
+                    .padding(padding)
+                    .consumeWindowInsets(padding)
+            ) {
+                LazyColumn(
+                    modifier = Modifier.weight(1f)
+                ) {
+                    if (state.lines.isEmpty()) {
+                        item {
+                            Spacer(Modifier.size(80.dp))
+                            Text(
+                                text = "Empty file",
```

Comment:
Shouldn't we use a string resource here?

Reply:
Yes, we should, but it's a debug screen, so I do not think it's worth it. One day `lint` or any other tool will complain about this issue and we will ignore it :)